docs/en/compute/AWS-EMR.md

```diff
@@ -45,7 +45,7 @@ $ aws emr create-default-roles
 ```
 2. The Alluxio bootstrap script is hosted in a publicly readable
 [S3 bucket](https://alluxio-public.s3.amazonaws.com/2.0.1/emr/alluxio-emr.sh).
-This bucket can also be accessed using it's S3 URI: `s3://alluxio-public/emr/alluxio-emr.sh`
+This bucket can also be accessed using it's S3 URI: `s3://alluxio-public/emr/2.0.1/alluxio-emr.sh`
```

Comment:
Can you double check the path? It looks like it's different from the http URL

Reply:
Good eye. Fixed now. Quick sweep shows that was the only one out of order.